fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1942,4 +1942,24 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = { enableFxSuggest },
         featureFlag = FeatureFlags.fxSuggest,
     )
+
+    /**
+     * Indicates if the user is shown new redesigned Toolbar UI.
+     */
+    var enableRedesignToolbar by lazyFeatureFlagPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_toolbar_use_redesign),
+        default = { false },
+        featureFlag = FeatureFlags.completeToolbarRedesignEnabled,
+    )
+
+    /**
+     * Indicates if the user is shown incomplete new redesigned Toolbar UI components and behaviors.
```

Comment:
@HarrisonOg @mavduevskiy I am wondering why we would want to have a separate settings for this? For developers, we would be able to see the WIP at any time by going to the secret settings with a single pref. At the moment, I would imagine the `enableRedesignToolbar` would also just as well represent the WIP toolbar redesign.